SDL/include/videoStream.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef VIDEOSTREAM_H
+#define VIDEOSTREAM_H 
+#include "threadClass.hpp"
```

Comment:
Nitpicky, but I think a empty line between 
`#define VIDEOSTREAM_H`
and 
`#include "threadClass.hpp"` 
would look nicer.